templates/test/cluster-template-prow-single-master.yaml

```diff
@@ -1,397 +0,0 @@
-apiVersion: cluster.x-k8s.io/v1alpha3
```

Comment:
To clarify, we can live without this file and control number of ctpl/workers from the go code itself using the vars from environment.sh?

Reply:
> To clarify, we can live without this file and control number of ctpl/workers from the go code itself using the vars from environment.sh?

Have the same concern. It would be nice if we can have that

Reply:
Yes. We can control the number of ctpl and workers using ControlPlaneMachineCount and WorkerMachineCount fields in `ConfigClusterInput` in the go code. No need to modify environment.sh either, except for setting NUM_NODES high enough to accomodate all the machines.